app/views/early_career_payments/claims/_ineligibility_generic.html.erb

```diff
@@ -1,5 +1,9 @@
 <p class="govuk-body">
-  You are not eligible for any additional payments.
+  You are not eligible for an
+  <%= link_to("early-career payments", EarlyCareerPayments.eligibility_criteria_url, class: "govuk-link") %>
+  or a
+  <%= link_to("levelling up premium payments", LevellingUpPremiumPayments.eligibility_criteria_url, class: "govuk-link") %>.
```

Comment:
Looking at the ticker more closely, I believe we need to use `eligibility_page_url` rather than `eligibility_criteria_url` here.